src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -422,6 +422,14 @@ robocopy /S /NP %EXECUTION_DIR%int\%(Identity)\ %EXECUTION_DIR%native\
       <PostExecutionTestCommandLines Include="move $(UAP_Results_Path)$(XunitTestAssembly).xml .\$(XunitResultsFileName)" />
     </ItemGroup>
 
+    <!-- If the PreExecutionTestScript property is set, then it should be set to the full path to a script that will be directly incorporated
+         into the generated runtests.cmd script, immediately before the test is run. This can be used to set a number of JIT stress modes,
+         for example.
+     -->
+    <ItemGroup Condition="'$(PreExecutionTestScript)'!=''">
+      <TestCommandLines Include="$([System.IO.File]::ReadAllText('$(PreExecutionTestScript)'))" />
```

Comment:
Don't know what this will look like when it puts it into the script will this be all one line? If so we might consider ReadAllLines if that helps.